client/lib/plugins/sanitize-section-content.js

```diff
@@ -1,7 +1,33 @@
-import { isEnabled } from '@automattic/calypso-config';
 import { addQueryArgs } from '@wordpress/url';
 import { filter } from 'lodash';
 import validUrl from 'valid-url';
+import { allowedTags, customTags } from './allowed-tags';
+
+let root = 'undefined' !== typeof window && window;
+let parser;
+
+/**
+ * Replace global root object with compatible one
+ *
+ * This is need in order to sanitize the content on the server.
+ *
+ * @param {object} newRoot window-like object to use as root
+ */
+export function overrideSanitizeSectionRoot( newRoot ) {
+	root = newRoot;
```

Comment:
One concern (but I'm not sure if it's valid) is whether this root and parser will be shared among multiple requests, and then whether that's problematic. (given it's a global value). Should we instead encapsulate it somehow?

I think I may just misunderstand what the root does :)

Reply:
It's a stand-in for the `window` object. On the server, we want to replace it with the `jsdom` version.

Reply:
The `root` property is overwritten on server start. Also, a `DOMParser` instance is created and shared between requests since we're only using its `parseFromString` method.

Client-side should continue to use `window` and create a new `DOMParser` instance each time, as before.

Reply:
Makes sense, thank you for clarifying 👍